### ISSUE TITLE
Auto-shift causes 45% CPU of kernel_task

### ISSUE THREAD

**Issue by m-emelchenkov** [aonez/NightShiftPatcher#10]
I use High Sierra on MBP 17" Late 2011. The app works fine, thank you. But, scheduled night shift causes 45% CPU of kernel_task right after switching, and does not fall off until manually turn it off and turn on again. Have you faced a such issue?

**Comment by aonez** [comment]
Not really, never had this issue. It’s been long since I don’t use night shift in an unsupported Mac anyway... I’ll try to check it later.

**Comment by m-emelchenkov** [comment]
Thank you very much. I'll be glad if you come across any relevant information regarding this issue (because I found nothing myself). I really enjoy Night Shift mode on my unsupported Mac.

**Comment by aonez** [comment]
@m-emelchenkov, just checked on a MacBook Air Mid 2011 running High Sierra 10.13.6, found no issue, enabling Night Shift manually or using the schedule (custom and sunset to sunrise).

I really don't know what the problem might be for you. Try creating a new user account, see if that helps.

**Comment by m-emelchenkov** [comment]
@aonez Yes, I tried even on a fresh install, it does not matter. Air 2011 has only integrated GPU, MBP also has AMD GPU. I have not tried Nigh Shift on Intel, I always work on AMD.

**Comment by aonez** [comment]
It worked on previous versions? Like in 10.12?

**Comment by m-emelchenkov** [comment]
I don't remember, unfortunately, I have not used 10.12 for a very long time. I can say for sure it has the same issue on different versions of 10.13.x.

**Comment by aonez** [comment]
I don't have any MacBook similar to yours, so I can't check :(

**Comment by m-emelchenkov** [comment]
Anyway, thanks for the willingness to help!
